### DISCUSSION TITLE
[V3] wire:navigate autofocus stop focusing after N navigate

### DISCUSSION THREAD

**Discussion by DanielSpravtsev** [livewire/livewire#7067] [7. Bugs (archived)]
### Livewire version

3.0.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create two pages with wire:navigate navigation between them
2. Place an `<input type="text" autofocus/>` at any of these pages
3. Navigate multiple times between pages using wire:navigate
4. Focus working only on first wire:navigate after manual page refresh. Focus stop working after 2+ wire:navigates 

I have only one though about it: autofocus function runs before input becomes available on the page after render

Will update discussion if I'll find more time to test it. 

### Additional information

Also leaving a link to the function that handle it
https://github.com/livewire/livewire/blob/07de2790f0aaf47c53f3a8cf942139a54b4609d8/js/plugins/navigate/index.js#L161

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by yassinetrbls** [comment]
Same issue on my side !

**Comment by davidwhicker** [comment]
same issue

**Comment by andrisunardi** [comment]
same issue here

**Comment by inmanturbo** [comment]
Yes I've faced this issue before and used alpine with 'trap' to work around it.

**Reply by bernhardh** [reply]
Can you please give me an example, how you have done this?

**Reply by inmanturbo** [reply]
I'm on my phone but, something like this:

```blade
<body class="font-sans antialiased" x-data="{ trapped: true }" x-on:keydown="trapped = false" x-on:mousemove="trapped = false">
 ....
    <div class="col-span-6 sm:col-span-4">
        <x-label for="amount" value="{{ __('Amount') }}" />
        <x-input id="amount" class="block w-full mt-1" type="text" wire:model="data.amount"  x-trap="trapped"/>
        <x-input-error for="data.amount" class="mt-2" />
    </div>
....
</body>
```

**Comment by mauricekindermann** [comment]
Better solution that I've started using as a fallback.

```html
<input type="text" x-data x-init="$nextTick(() => $el.focus())" />
```

It would be nice if Livewire could handle this itself.

**Reply by jeffchown** [reply]
Just ran into this for the first time.  Thanks for the workaround @mauricekindermann !

Anyone know if this has been addressed in Livewire in recent releases? @joshhanley ?
